src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (robot[key] in invertedRobot) {
```

Comment:
The check `robot[key] in invertedRobot` is correct for determining if a property exists in an object. However, it's important to note that using `in` will also return true for inherited properties. To ensure that only own properties are considered, you might want to use `Object.prototype.hasOwnProperty.call(invertedRobot, robot[key])` or `invertedRobot.hasOwnProperty(robot[key])`.